### PR TITLE
Submarine Bucket Use Toggle

### PR DESCRIPTION
**Pull request by R-Jimenez**
Feature outlined in #354.

### CODE REVIEW

File: config.ini [hunk 1]
```diff
@@ -63,6 +63,7 @@ Enabled = True
 EnabledSubs = ss
 ReplaceLimit =
 FatigueSwitch = False
+UseBuckets = false
```
Comment (mrmin123) [inline config.ini]: Please change `false` to `False` (`ConfigReader` can deal with `false`, but to keep in-line with Python standards). If you could also update `config_detailed.ini` with this new parameter and add a description of what it does that'd be great.

**Review by mrmin123 — CHANGES_REQUESTED** [review]
Requested some changes in `config.ini` and `config_detailed.ini`. Code changes looks good but I'd also like to test it a bit before merging.

File: kancolle_auto.sikuli/combat.sikuli/combat.py [hunk 2]
```diff
@@ -490,20 +491,62 @@ def go_repair(self):
...
-                            bucket_use = True
+                            if (not self.submarine_switch) or (self.submarine_switch and self.submarine_switch_use_buckets):
+                                # Use buckets on subs if submarine handling is disabled, or buckets use is allowed
+                                log_success("Repair time too long... using bucket!")
+                                self.kc_region.click('repair_bucket_switch.png')
+                                self.next_sortie_time_set(0, 0)
+                                bucket_use = True
+                            else:                                
+                                # Submarine switch is enabled, but bucket use is disallowed                                
+                                if global_regions['repair_panel'].exists(Pattern('ship_class_ss.png').similar(0.8)) or global_regions['repair_panel'].exists(Pattern('ship_class_ssv.png').similar(0.65)):
```
Comment (mrmin123) [inline kancolle_auto.sikuli/combat.sikuli/combat.py]: There's a `CLASS_SIMILARITY` constant defined at the beginning of the file. Was this not sufficient for `ship_class_ss` and `ship_class_ssv`? I'd like to set the threshold for all ship class matching to a constant if at all possible.

Comment (R-Jimenez) [reply kancolle_auto.sikuli/combat.sikuli/combat.py]: Will have to lower CLASS_SIMILARITY to about .65 if that's the case. I tried a couple environments but the SSV icon is tricky to match, especially with dmg effects.

**Comment by mrmin123** [comment]
Thanks for the update. As an FYI, I am going to defer feature additions until post-event. I'd also like to look into the CLASS_SIMILARITY value a bit more before merge. Depending on the needed values the approach to the constant might have to change.

**Comment by R-Jimenez** [comment]
Roger that. o7 I feel like having a high CLASS_SIMILARITY is worth it, since only one or two classes are tricky enough to warrant a lower setting for accurate searching. Why not just have two tiers? CLASS_SIMILARITY and CLASS_SIMILARITY_LOW/CLASS_SIMILARITY_SSV?

**Comment by mrmin123** [comment]
@R-Jimenez please separate out the `0.8` and `0.65` thresholds to additional globals at top of `combat.py` as `CLASS_SIMILARITY_SS` and `CLASS_SIMILARITY_SSV` and reference those in the code. It'll help during refactor so I can keep track of globals.

**Comment by R-Jimenez** [comment]
Changes pushed.

**Comment by mrmin123** [comment]
@R-Jimenez It looks like some unrelated changes got pushed with your latest commits. Please revert those, merge in master, and repush the branch.

**Comment by R-Jimenez** [comment]
Sorry about that. Git had some confusion with the branches and pulled files I thought were not committed. Should be much cleaner now

**Pull request merged**